interceptor/src/io/pedestal/interceptor/chain.clj

```diff
@@ -30,7 +30,8 @@
   (get interceptor :name (pr-str interceptor)))
 
 (defn- throwable->ex-info [^Throwable t execution-id interceptor stage]
-  (ex-info (str "Interceptor Exception: " (.getMessage t))
+  (ex-info (str "Interceptor Exception: " (or (.getMessage t)
```

Comment:
Is the intent to also fall-back to the exc class name  if user supplied an empty message?
e.g: if the user did `(SomeException. "")` ?
In which case, perhaps need to do something like `(non-empty (.getMessage t))`.
Otherwise, looks fine.